src/components/button-toggle-group/button-toggle-group.component.tsx

```diff
@@ -106,50 +115,65 @@ const ButtonToggleGroup = ({
     `\`ButtonToggleGroup\` only accepts children of type \`${ButtonToggle.displayName}\``
   );
 
+  if (name && !deprecateNameWarnTriggered) {
+    deprecateNameWarnTriggered = true;
+    Logger.deprecate(
+      `The \`name\` prop in \`ButtonToggleGroup\` component is deprecated and will soon be removed. It does not provide any functionality
+      since the component can no longer be used in an uncontrolled fashion.`
+    );
+  }
+
+  const [pressedButtonValue, setPressedButtonValue] = useState<string | null>(
+    null
+  );
+
+  const fieldHelpContent = fieldHelp ? (
+    <FieldHelp labelInline={labelInline} labelWidth={labelWidth}>
+      {fieldHelp}
+    </FieldHelp>
+  ) : null;
+
   return (
     <TooltipProvider helpAriaLabel={helpAriaLabel}>
-      <InputGroupBehaviour>
-        <FormField
-          useValidationIcon={validationOnLabel}
-          label={label}
-          labelHelp={labelHelp}
-          labelSpacing={labelSpacing}
-          fieldHelp={fieldHelp}
-          fieldHelpInline={fieldHelpInline}
-          labelInline={labelInline}
-          labelWidth={labelWidth}
-          labelAlign={labelAlign}
-          data-component={dataComponent}
-          data-role={dataRole}
-          data-element={dataElement}
-          id={id}
-          {...validationProps}
-          {...filterStyledSystemMarginProps(props)}
-        >
+      <ButtonToggleGroupContext.Provider
+        value={{
+          onButtonClick: setPressedButtonValue,
+          pressedButtonValue: value || pressedButtonValue,
+          onBlur,
+          onChange,
+          name,
+        }}
+      >
+        <StyledButtonToggleGroupWrapper labelInline={labelInline}>
+          {label && (
```

Comment:
What is the reason for not using the `FormField` here?

Reply:
It seems fine using FormField now I go back to it - I can just leave off the `validationProps` etc. It means I don't have to recreate the styling/placement of things like the labels and fieldhelp - not sure why I didn't use it before but I'll restore its use to this component as I don't see a good reason not to have it.